aws_lambda_powertools/utilities/parser/models/transfer_family.py

```diff
@@ -5,8 +5,28 @@
 
 
 class TransferFamilyAuthorizer(BaseModel):
-    username: str
-    password: Optional[str] = None
-    protocol: Literal["SFTP", "FTP", "FTPS"]
-    server_id: str = Field(..., alias="serverId")
-    source_ip: IPvAnyAddress = Field(..., alias="sourceIp")
+    username: str = Field(
+        description="The username of the user attempting to authenticate.",
+        examples=["bobusa", "john.doe", "sftp-user-123", "data-transfer-user"],
+    )
+    password: Optional[str] = Field(
+        default=None,
+        description="The password for authentication.",
+        examples=["<password>", "<user-password>", None],
+    )
+    protocol: Literal["SFTP", "FTP", "FTPS"] = Field(
+        description="The protocol used for the connection.",
+        examples=["SFTP", "FTPS", "FTP"],
+    )
+    server_id: str = Field(
+        ...,
+        alias="serverId",
+        description="The server ID of the Transfer Family server.",
+        examples=["s-abcd123456", "s-1234567890abcdef0", "s-example123"],
+    )
+    source_ip: IPvAnyAddress = Field(
+        ...,
+        alias="sourceIp",
+        description="The IP address of the client connecting to the Transfer Family server.",
+        examples=["192.168.0.100", "10.0.0.50", "127.0.0.1", "203.0.113.12"],
```

Comment:
Please remove examples on in this case. It potentially will trigger some security pipeline in customers.